extlinks/links/management/commands/linkevents_archive.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+import glob, gzip, logging, math
+
+from django.core import serializers
+from django.core.management import BaseCommand
+from django.db import IntegrityError
+
+from extlinks.links.models import LinkEvent
+
+logger = logging.getLogger("django")
+chunk = 100000
+
+
+class Command(BaseCommand):
+    help = "dump + delete or load LinkEvents for a given year"
+
+    def dump(self, year):
```

Comment:
Please add a docstring to both functions for future reference :)

Reply:
done